packages/desktop-client/src/components/budget/MobileTable.tsx

```diff
@@ -1,11 +1,18 @@
-import React from 'react';
+import React, { type ReactNode } from 'react';
+
+import { type CSSProperties } from 'glamor';
```

Comment:
cc @joel-jeremy

Reply:
We should use React's CSSProperties here instead of glamor's

Reply:
Thanks for the input, I'll fix that.